bin/run_ess_sensors.py

```diff
@@ -21,12 +21,40 @@
 import asyncio
 import logging
 
-from lsst.ts.ess import sensors
+from lsst.ts.ess.sensors import SocketServer
 
 logging.basicConfig(
     format="%(asctime)s:%(levelname)s:%(name)s:%(message)s",
     level=logging.DEBUG,
 )
 
 
-asyncio.run(sensors.SocketServer.main())
+async def main():
+    """Main method that, when executed in stand alone mode, starts the socket
+    server.
+
+    The SocketServer automatically stops once the client exitst and at that
+    moment this script will exit as well.
+    """
+    logging.info("main method")
+    # An arbitrarily chosen port. Nothing special about it.
+    port = 5000
+    logging.info("Constructing the sensor server.")
+    # Simulation mode 0 means "connect to the real sensors."
+    # Set simulation_mode to 1 to enable simulation mode and connect to a mock
+    # sensor.
+    srv = SocketServer(port=port, simulation_mode=0)
+    logging.info("Starting the sensor server.")
+    await srv.start()
```

Comment:
`start` sounds like it should exit once the socket server is accepting connections, but given this code I assume it actually waits fore more than that -- I'm not sure how much more.

I suggest making `start` exit as soon as the server is accepting connections. It seems more natural then waiting for ...what? Also, should you decide to support random ports, this gives you perfect way to report which port was chosen:
```
await srv.start()
print(f"port=...")
await asyncio.Future()
```

Reply:
See my previous comment about the random port. Actually, `start` does exit as soon as it starts accepting connections.